lib/msf/core/payload/windows/reverse_winhttps.rb

```diff
@@ -114,6 +97,23 @@ def required_space
     space
   end
 
+  #
+  # Get the SSL hash from the certificate, if required.
+  #
+  def get_ssl_cert_hash
```

Comment:
Yeah it is a bit of copy pasta. I wasn't totally sure how I should deal with this given that it deals with the datastore stuff directly.

Happy to shove it in a mixin.